src/Mailchimp/Manager.php

```diff
@@ -54,16 +54,10 @@ public function editMember(Adherent $adherent, AdherentChangeCommandInterface $m
         );
 
         if ($result) {
-            $this->logger->info(sprintf('Mailchimp member "%s" has been edited', $adherent->getUuidAsString()));
-
             // Active/Inactive member's tags
-            $result = $this->driver->updateMemberTags(
+            $this->driver->updateMemberTags(
                 $requestBuilder->createMemberTagsRequest($adherent->getEmailAddress(), $message->getRemovedTags())
             );
-
-            if ($result) {
-                $this->logger->info(sprintf('Mailchimp member "%s" tags have been updated', $adherent->getUuidAsString()));
```

Comment:
Why does logs are removed ? Isn't it dangerous to loose thoose informations in case of bug ?